src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -324,6 +333,18 @@
     /** Actions to add to the {@link FlowStartNode}. */
     transient final List<Action> flowStartNodeActions = new ArrayList<Action>();
 
+    /** If true, pipeline is forbidden to resume even if it can. */
+    public boolean isResumeBlocked() {
+        return resumeBlocked;
+    }
+
+    public void setResumeBlocked(boolean resumeBlocked) {
```

Comment:
Who calls this? It seems only test code for now? What is the expected UX?

Reply:
Downstream - used in WorkflowJob